### PR TITLE
MMCA-4996 Change filerole value from CashStatement to CDSCashAccount

### PR DESCRIPTION
**Pull request by harshare**

### CODE REVIEW

**Review by AdamCoops — APPROVED** [review]

File: test/connectors/SdesConnectorSpec.scala
Comment (tyagikumarvineet) [inline test/connectors/SdesConnectorSpec.scala:1]: Could you please replace CashStatement with CDSCashAccount at line 103, 115 and 129?

File: app/config/AppConfig.scala [hunk 1]
```diff
@@ -79,7 +79,7 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
 
   def requestedStatements(fileRole: FileRole): String = {
     fileRole match {
-      case FileRole.CashStatement => s"$requestedStatements${fileRole.featureName}"
+      case FileRole.CDSCashAccount => s"$requestedStatements${fileRole.featureName}"
```
Comment (tyagikumarvineet) [inline app/config/AppConfig.scala:82]: Could you please write the test for this change?

File: test/models/FileRoleSpec.scala [hunk 2]
```diff
@@ -25,8 +25,8 @@ class FileRoleSpec extends SpecBase {
   "FileRole" should {
 
     "return CashStatement" in {
```
[the remark this answers is not in the document]
Comment (harshare) [reply test/models/FileRoleSpec.scala:27]: Updated test description to use CDSCashAccount

File: test/connectors/SdesConnectorSpec.scala
Comment (harshare) [reply test/connectors/SdesConnectorSpec.scala:1]: Updated test description to use CDSCashAccount

File: app/config/AppConfig.scala (hunk 1 above)
Comment (harshare) [reply app/config/AppConfig.scala:82]: Added test for this change

**Review by tyagikumarvineet — APPROVED** [review]

**Review by tyagikumarvineet — APPROVED** [review]
Build failing due to resource exhaustion

**Pull request merged**